noxfile.py

```diff
@@ -797,6 +797,65 @@ def integration_tests_pika(
                 kill_process_and_clean_outputs(temp_file, "uvicorn", session)
 
 
+@nox.session()
+@nox.parametrize(
+    "python,psycopg2_version",
+    [
+        (python, psycopg2_version)
+        for python in python_versions()
+        for psycopg2_version in dependency_versions_to_be_tested(
+            python=python,
+            directory="psycopg2",
+            dependency_name="psycopg2",
+        )
+    ],
+)
+def integration_tests_psycopg2(
+    session,
+    python,
+    psycopg2_version,
+):
+    with TestedVersions.save_tests_result(
+        "psycopg2", python, "psycopg2", psycopg2_version
+    ):
+        install_package("psycopg2", psycopg2_version, session)
+
+        session.install(".")
+
+        temp_file = create_it_tempfile("psycopg2")
+        with session.chdir("src/test/integration/psycopg2"):
+            session.install("-r", OTHER_REQUIREMENTS)
+
+            try:
+                session.run(
+                    "sh",
+                    "./scripts/start_uvicorn",
```

Comment:
This is a database. Why do you need this extra uvicorn with you?
just copy what we did for redis - this is much simpler test, with less dependencies, thus less flaky!

Reply:
great point, done 👌